packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -425,7 +448,29 @@ contract OPContractsManager_Upgrade_TestFails is OPContractsManager_Upgrade_Harn
 
         vm.expectRevert("Ownable: caller is not the owner");
         DelegateCaller(delegateCaller).dcForward(
-            address(opcm), abi.encodeCall(IOPContractsManager.upgrade, (proxyAdmin, opChains))
+            address(opcm), abi.encodeCall(IOPContractsManager.upgrade, (proxyAdmin, opChainConfigs))
+        );
+    }
+
+    function test_upgrade_permissionedPrestateNotSet_reverts() public {
+        vm.etch(upgrader, vm.getDeployedCode("test/mocks/Callers.sol:DelegateCaller"));
```

Comment:
nit, should this just be de-duped and etched in setup where the `upgrader` address is defined?